Makefile

```diff
@@ -261,6 +261,23 @@ dist/module_webworker_dev.js: src/templates/module_webworker.js
 dist/webworker_dev.js: src/templates/webworker.js
 	cp $< $@
 
+
+# Prepare the dist directory for the release by removing unneeded files
+.PHONY: clean-dist-dir
+clean-dist-dir:
+	# Remove snapshot files
+	rm dist/makesnap.mjs
+	rm dist/snapshot.bin
+	rm dist/module_test.html dist/test.html
+	# TODO: Remove webworker.js too? Would require updating the docs I think.
```

Comment:
Sounds good to me. I think it is mostly for reference and not used anywhere except for the docs.